### ISSUE TITLE
Sending ETH doesn't work

### ISSUE THREAD

**Issue by mhluongo** [tallycash/extension#2010]
![Screenshot from 2022-08-23 14-06-17](https://user-images.githubusercontent.com/427505/186232388-7fdf3a52-cbdc-4ac6-8682-7f41f1aec5e1.png)

Sending ETH often returns to the asset page rather than the sign page. This behavior appears to depend on the recipient address... making me suspect #1900

**Comment by 0xDaedalus** [comment]
Would you mind pasting an address here that you've seen `send` fail for?

**Comment by mhluongo** [comment]
Yep! I was using `0x85eee30c52b0b379b046fb0f85f4f3dc3009afec`, the KEEP token address. Note this was on `main` prior to the merge of #1993

**Comment by 0xDaedalus** [comment]
No luck on a quick fix but one thing that is happening is we're getting `{code: -32000, message: "execution reverted"}` back from alchemy when calling `eth_estimateGas` with the tx

**Comment by hyphenized** [comment]
Could this be related to contracts that have their receive function set to revert transactions or lack a receive function? Sending ETH to BNB `0xB8c77482e45F1F44dE1745F52C74426C631bDD52` works 🤔

**Comment by 0xDaedalus** [comment]
Yes - I do think that could be the case here.

My Go is a bit rusty - but here's what I think is going on:

When GETH tries to [estimates gas](https://github.com/ethereum/go-ethereum/blob/2c5648d8915dc6f4cd822a379819d774beba0dc5/internal/ethapi/api.go#L1027) - it [simulates the transaction](https://github.com/ethereum/go-ethereum/blob/2c5648d8915dc6f4cd822a379819d774beba0dc5/internal/ethapi/api.go#L1101) - and eventually [returns an error](https://github.com/ethereum/go-ethereum/blob/2c5648d8915dc6f4cd822a379819d774beba0dc5/internal/ethapi/api.go#L1136) if the all attempts at executing the transaction fail.

**Comment by mhluongo** [comment]
The `payable` / fallback idea is definitely it @hyphenized. The right behavior here is to show that the tx has failed though... especially if I signed it first.

**Comment by hyphenized** [comment]
Can we check if the contract has a receive function and otherwise send users to the contract interaction screen?

**Comment by Shadowfiend** [comment]
Gas estimation reverts deserve their own admonition for the user, fwiw. 

Do you see a snackbar indicating the tx failed?
